### PR TITLE
Fix wrong groups name and permissions names in the documentation.

### PR DESCRIPTION
**Pull request by Natim**
r? @leplatrem @almet

### CODE REVIEW

**Comment by aegaas** [comment]
Yes this is great! Thank you for updating this

File: docs/tutorials/permission-setups.rst [hunk 1]
```diff
@@ -80,16 +80,16 @@ The following objects are created:
...
-+--------------------------+--------------+-----------------------------------+
++--------------------------+--------------+------------------------------------------+
+| Object                   | Permissions  | Principals                               |
++==========================+==============+==========================================+
+| Bucket ``companywiki``   | ``write``    | ``fxa:<wiki administrator id>``          |
++--------------------------+--------------+------------------------------------------+
+| Group ``employees``      | ``write``    | ``/buckets/companywiki/groups/managers`` |
++--------------------------+--------------+------------------------------------------+
+| Collection ``articles``  | ``write``    | ``group:employees``                      |
+|                          |              | ``group:managers``                       |
```
Comment (leplatrem) [inline docs/tutorials/permission-setups.rst]: Looks like you forget these two

Comment (Natim) [inline docs/tutorials/permission-setups.rst]: Good catch !

**Comment by leplatrem** [comment]
r+wc

**Pull request merged**